[BIP-666] CoWAMM POL

Xeonus opened the pull request:
- transfer funds to LM multisig to seed CoWAMM pools with Balancer POL
- ~~200803.3 BAL~~ 213_675.21 BAL
- ~~375000 USDC~~ 125_000 USDC
- ~~38.121 WETH~~ 118.26 WETH

[Proposal](https://forum.balancer.fi/t/bip-666-provide-balancer-dao-pol-to-cowamm-pools/5895)

gosuto-inzasheru reviewed (changes requested):
this does not add up for me. reqs for balancer i distill from bip628:
- swap $500k bal for $500k cow
- (earmark $1m bal to swap for $1m cow at $50m tvl)
- (earmark $1m bal to swap for $1m cow at $100m tvl)
- $250k bal for incentives
- contribute 25% of $1m usdc/weth pool
- $500k bal/weth pool

so this proposed payload deviates from that in the following ways:
- no $500k bal to cow treasury swap
- why transfer usdc to seed bal/weth pool?

Xeonus commented:
@gosuto-inzasheru 

- We only want to seed POL with this proposal as indicated with the title. I added a specification line in the BIP that no treasury swap tokens are currently considered
- As per [BIP-628](https://forum.balancer.fi/t/bip-628-balancer-cow-protocol-strategic-partnership-launching-cowamm-pools/5813) we need to seed the following: $250k to the USDC to the USDC/WETH pool, $500k to the BAL/WETH pool
- I agree that MikeB made a mistake in transferring too much USDC instead of WETH, fixed in follow-up commit

gosuto-inzasheru reviewed (changes requested):
ok sure only pol and incentives in this payload, so:
- $250k bal for incentives
- $125k usdc for usdc/weth
- $125k weth for usdc/weth
- $250k bal for bal/weth
- $250k weth for bal/weth

($250k+$250k)/2.34 = 213_675 bal
($125k+$250k)/3169 = 118.33385926 weth

am i missing something? @Xeonus how did you get to 181weth?
